src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import './variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
The `body` style rules seem to be responsible for centering the `.calendar` on the page. It would be better to move these rules to the `.calendar` selector. This way, the `.calendar` will be centered regardless of where it is placed in the HTML structure.

Reply:
it is against the condition